pages/profile/index.js

```diff
@@ -0,0 +1,313 @@
+import { useState } from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+import useLocalStorageState from "use-local-storage-state";
+import { useRouter } from "next/router";
+
+const avatars = [
+  "/avatars/avatar1.png",
+  "/avatars/avatar2.png",
+  "/avatars/avatar3.png",
+];
+
+export default function UserProfile({
+  isEditing,
+  setIsEditing,
+  successMessage,
+  setSuccessMessage,
+}) {
+  const [profile, setProfile] = useLocalStorageState("userProfile", {
+    defaultValue: {
+      firstName: "",
+      lastName: "",
+      avatar: "/avatars/avatar1.png",
```

Comment:
since you want to use one element of your `avatars` here, you can use the variable instead of duplicating the link to the image: `avatar: avatars[0]`